storage/storage.go

```diff
@@ -0,0 +1,16 @@
+package storage
+
+import (
+	"context"
+
+	accountspb "github.com/openbank/gunk/gunk/v1/accounts"
+)
+
+// AccountStore provides storage operation for account resource.
+type AccountStore interface {
+	GetAccount(ctx context.Context, id string) (*accountspb.Account, error)
+	GetAccounts(ctx context.Context, nextID string) ([]*accountspb.Account, bool, error)
```

Comment:
Maybe use `ListAccount` instead :thinking:

Reply:
Hm I don't really see the point of wrapping the slice and the boolean here. Having 3 return values doesn't seems to be too much. 
I can add names if you're concerned about readability ?